src/label_nodes/CCLabelBMFont.cs

```diff
@@ -271,25 +274,30 @@ public CCLabelBMFont(string str, string fntFile, float width, CCTextAlignment al
         {
         }
 
-        public CCLabelBMFont(string str, string fntFile, float width, CCTextAlignment hAlignment, CCVerticalTextAlignment vAlignment, 
+        public CCLabelBMFont(string str, string fntFile, float width, CCTextAlignment hAlignment, CCVerticalTextAlignment vAlignment,
             CCPoint imageOffset, CCTexture2D texture)
             : this(str, fntFile, new CCSize(width, 0), hAlignment, vAlignment, imageOffset, texture)
         {
         }
 
-        public CCLabelBMFont(string str, string fntFile, CCSize dimensions, CCTextAlignment hAlignment, CCVerticalTextAlignment vAlignment, 
+        public CCLabelBMFont(string str, string fntFile, CCSize dimensions, CCTextAlignment hAlignment, CCVerticalTextAlignment vAlignment,
             CCPoint imageOffset, CCTexture2D texture)
         {
             InitCCLabelBMFont(str, fntFile, dimensions, hAlignment, vAlignment, imageOffset, texture);
         }
 
-        protected void InitCCLabelBMFont(string theString, string fntFile, CCSize dimensions, CCTextAlignment hAlignment, CCVerticalTextAlignment vAlignment, 
+        protected void InitCCLabelBMFont(string theString, string fntFile, CCSize dimensions, CCTextAlignment hAlignment, CCVerticalTextAlignment vAlignment,
             CCPoint imageOffset, CCTexture2D texture)
         {
             Debug.Assert(FontConfiguration == null, "re-init is no longer supported");
             Debug.Assert((theString == null && fntFile == null) || (theString != null && fntFile != null),
                 "Invalid params for CCLabelBMFont");
 
+            if (!((theString == null && fntFile == null) || (theString != null && fntFile != null)))
+            {
+                Debug.WriteLine("ERROR");
```

Comment:
I remember now that I left this here because I was debugging with a break point here -- there was a test case where the entire application crashed in some test case where I made it to this line.

This is exactly the Assert above but we need to fail InitCCLabelBMFont and not continue after the Assert because bad things happen.